cohorts/survival.py

```diff
@@ -76,6 +80,38 @@ def plot_kmf(df,
         print_as_title: bool, optional, whether or not to print text
           within the plot's title vs. stdout, default False
     """
+    # if strata not None, call plot_kmf for each of the strata
+    if strata_col:
+        # want this at the top of the function!
+        arglist = locals() ## save input args for reuse
```

Comment:
Using `locals()` feels a little hackish and unnecessary and easy to screw up of someone moves it after the first line; I'd prefer just passing the parameters through.

Also, I think it's probably not good to modify the `locals()` `dict` as you're doing with `arglist`?

https://docs.python.org/3/library/functions.html#locals
> modifications to the default locals dictionary should not be attempted.